governance/SDK.md

```diff
@@ -0,0 +1,172 @@
+## SDK Community
+
+The community is organized as follows:
+
+- Every SDK has its own
+  [Github Team](https://github.com/orgs/cloudevents/teams), for example
+  `sdk-go-maintainers` is the group of mantainers for
+  [cloudevents/sdk-go project](https://github.com/cloudevents/sdk-go)
+- The union of all the `sdk-*-maintainers` assembles the _sdk maintainers_ group
+
+## New maintainers
+
+TBD
+
+## Ensuring projects health
+
+In order to ensure the health of the SDK projects, we define the following
+actions. These are crucial to show the commitment of the community to develop
+and support these libraries, which are an important part of the CloudEvents
+ecosystem.
+
+We define an SDK project _healthy_ if:
+
+1. It works with the latest version of the programming language
+2. It supports the latest versions of the integrated libraries/frameworks
```

Comment:
> Does this mean "dependencies are kept up to date"? If so, why not just say that?

That's not necessary true, in ecosystems like Java libraries could be bytecode backward compatibility. For example, if sdk-java depends on library-x v1.0, it could work with v1.1 or even with v2.0 without touching sdk-java.

Reply:
So I guess I'm just not clear on what this means.

Reply:
Not sure how i could rephrase it honestly :man_shrugging: